test/goldens/end2end/angular_components-0.10.0.json_report.md

```diff
@@ -41,6 +41,12 @@ See [package layout](https://dart.dev/tools/pub/package-layout#examples) guideli
 
 ### [~] 10/20 points: Supports 1 of 2 possible platforms (native, js)
 
+* Package not compatible with runtime native-aot
```

Comment:
Is there a case where `native-aot` and `native-jit` differs? Can we emit only one of them with the tag `native`?

Reply:
I think `dart:developer` is the one one: https://dart.dev/guides/libraries

I would prefer if we generally just say "Package is not compatible with the Dart native platform", and then only if we have a specific check for `dart:developer` then for that check say "Package is not compatible with the Dart native platform in AOT mode"

Reply:
Generally, we don't mention "AOT"-mode on pub.dev.. We just say that something is "native" if it works with "native-jit".

Reply:
This will be handled in a separate PR.